app/views/mailers/member_mailer/access_granted_manager_email.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= render partial: "mailers/email_template",
+locals: {
+  url:
+    (
+      if @namespace.type == "Group"
+        group_members_url(@namespace)
+      else
+        namespace_project_members_url(@namespace.parent, @namespace.project)
+      end
+    )
+} %>
+
+<%= yield :header_and_greeting %>
+
+<tr>
+  <td style="padding:12px 24px;">
+    <p style="color:#334155;line-height:24px;">
+      <%= t(
+        :"mailers.member_mailer.access_granted_manager_email.body_html",
```

Comment:
**nitpick (blocking):** Can you display the user that has been granted access like below:

`John Doe (john.doe@email.com)`

In this case email address is our unique identifier for a user so this is clearer on who has been granted access.

Reply:
I added the user's email to the body of the manager templates in 0aa9c9c2002ef512eae14db6f686036accdc337a. I left the single quotes around the actual group/project and user names for clarity. I hope that's ok? Did you want the user's email included in the manager email subject too?